geode-redis/src/main/java/org/apache/geode/redis/internal/gfsh/RedisCommandFunction.java

```diff
@@ -36,7 +36,8 @@ public CliFunctionResult executeFunction(FunctionContext<Boolean> context) {
     InternalCache cache = (InternalCache) context.getCache();
 
     if (!cache.getInternalDistributedSystem().getConfig().getRedisEnabled()) {
-      return new CliFunctionResult(context.getMemberName(), false, "Error: Redis is not enabled");
+      return new CliFunctionResult(context.getMemberName(), false,
+          "Error: APIs Compatible with Redis is not enabled");
```

Comment:
I don't think we should capitalize "Compatible".

"Error: APIs compatible with Redis are not enabled"